python/cudf/cudf/tests/test_text.py

```diff
@@ -170,3 +171,197 @@ def test_ngrams_tokenize(n, separator, expected_values):
 
     assert type(expected) == type(actual)
     assert_series_equal(expected.to_pandas(), actual.to_pandas())
+
+
+def test_character_tokenize_series():
+    sr = cudf.Series(
+        [
+            "hello world",
+            "sdf",
+            None,
+            "goodbye, one-two:three~four+five_six@seven#eight^nine",
```

Comment:
Can you throw in a multi-byte character in here as well. Like `é` for example.

Reply:
Added few more utf-8 chars